rocfft/PKGBUILD

```diff
@@ -1,32 +1,31 @@
 # Maintainer: Jakub Okoński <jakub@okonski.org>
 # Maintainer: Markus Näther <naetherm@cs.uni-freiburg.de>
 pkgname=rocfft
-_pkgver=3.3.0
-pkgver="$_pkgver"
+pkgver=3.3.0
 pkgrel=1
-pkgdesc="Next generation FFT implementation for ROCm"
+pkgdesc='Next generation FFT implementation for ROCm'
 arch=('x86_64')
 url="https://github.com/ROCmSoftwarePlatform/rocfft"
 license=('MIT')
-depends=('boost' 'fftw')
-makedepends=("hcc>=$pkgver" 'cmake')
-source=("https://github.com/ROCmSoftwarePlatform/rocFFT/archive/rocm-$_pkgver.tar.gz")
+depends=('hip-hcc')
```

Comment:
Should this just be `hip` so that `hip-nvcc` and `hip-clang` can work with it?

Reply:
It only works on the ROCm platform. For CUDA there is cuFFT, part of CUDA.

Reply:
I see kk